database/migrations/community/2024_03_30_000001_update_messages_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        if (Schema::hasColumn('messages', 'UserTo')) {
+            Schema::table('messages', function (Blueprint $table) {
+                $table->dropColumn('UserTo');
+            });
+        }
+
+        if (Schema::hasColumn('messages', 'UserFrom')) {
+            Schema::table('messages', function (Blueprint $table) {
+                $table->dropColumn('UserFrom');
+            });
+        }
+    }
```

Comment:
There are still references to these columns in the MigrateMessages command, which is no longer needed. Please remove it as part of this PR.

Reply:
Removed in latest.

It seems like we have a lot of dead code like this hanging around. At some point, we should probably sweep through and delete it all.